uberenv.py

```diff
@@ -464,10 +594,9 @@ def install(self):
         install_cmd = "spack/bin/spack "
         if self.opts["ignore_ssl_errors"]:
             install_cmd += "-k "
-        if not self.opts["install"]:
-            install_cmd += "dev-build -d {} -u {} ".format(self.pkg_src_dir,self.pkg_final_phase)
-        else:
-            install_cmd += "install "
+        install_cmd += "dev-build -d {0} ".format(self.pkg_src_dir)
+        if not self.opts["install"] and self.pkg_final_phase:
```

Comment:
This was brought in from Axom and has been decided to not be desired, though I would argue it will clean up some errors people will run into when your spack package doesn't have any phases (like bundle packages).

Reply:
What was the reasoning exactly? I know we discussed it for #17, and maybe it was simply to preserve the original behavior.

Reply:
So here is how I understand it and maybe we can reach an agreement.  We have two cases here:

1) Normal case (for uberenv at least):

We have a package which is a full package (like axom) where we want "uberenv axom" to install just the dependencies and stop at the final phase.  Then "uberenv --install axom" to install deps and axom itself.

2) My new introduced case which breaks the devtools away from main package. Because it is a spack bundlepackage it doesn't have phases. So i wanted "uberenv axomdevtools" to be the same as "uberenv --install axomdevtools" because there isn't a final phase to define.

Reply:
You could define a fake phase. (I know, what an elegant solution ...)

Reply:
I think the problem was that if final phase is undefined, then Uberenv defaults to install the package and not only it’s dependencies. And that would be a change of the workflow.